annual_reports/2024/index.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: page
+title: 5BFP 2024 transparency report
+---
+
+This report summarizes the activity of the Five Borough Fedi Project (5BFP) during calendar year 2024, including a high-level accounting of the organization’s financial income and expenditures. As a public-service organization that many people entrust with their time, attention, and money, 5BFP presents this report in an effort to show how it has applied its community’s investments over the past calendar year.
+
+5BFP president Jason McIntosh wrote and published this report in February 2025, based on 5BFP’s internal financial records. All monetary amounts that appear in this report are expressed in U.S. dollars.
+
+## Summary
+
+5BFP incorporated with New York State on February 23, 2024, under the full legal name "Five Borough Fedi Project, Inc." We received 501(c)(3) charitable nonprofit recognition from the IRS on April 22.
+
+During calendar year 2024, 5BFP spent a bit less than $1,500, mostly on paying for the various hosting services that power Masto.NYC. We started to use and pay for these services only in August 2024, and the relatively modest expenditure reflects this late start.
+
+5BFP operations have largely been covered by a generous corporate matching program, associated with a small number of donors. This has given the organization the good fortune of a relatively worry-free launch, though we do need to follow up to reduce its financial dependence on a single income source.
+
+## Program activity
+
+In 2024, the only program of 5BFP was the care and maintenance of the Masto.NYC Mastodon server. Long-term stewardship of Masto.NYC was the founders' primary reason for incorporating 5BFP. As of the end of the year, Masto.NYC remains the organization's sole focus and priority.
+
+Masto.NYC was originally launched as a passion project in late 2022, well before the incorporation (or even the ideation) of 5BFP. The Masto.NYC service continued uninterrupted through the formation of 5BFP and the subsequent migration of Masto.NYC infrastructure from physical servers in an East Village apartment to virtual servers in a Digital Ocean datacenter, located elsewhere in New York City.
+
+## Financial summary
+
+We took in $4,174.68 and spent $1,453.39, for a net gain of $2,721.29. Since 2024 was the year that the organization, $2,721.29 also reflects the amount of money that 5BFP had in its bank account at year's end.
+
+### Income
+
+<img src="income.png" style="float:right; width: 220px" />
+
+5BFP took in $4,174.68 during 2024:
+
+* $3,745.00 came from a corporate charity program, with employees of that corporation using their philanthropy employment benefit to direct donations to 5BFP.
```

Comment:
...I think we should be more clear here, and say "with one employee of that corporation". I want to be as clear as possible and relay the (high amounts of) risk appropriately.

Reply:
Done.